### PR TITLE
[#5064] docs(spark): support s3 storage for spark hive connector

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?

add document about support s3 storage for spark hive connector

### Why are the changes needed?

Fix: #5064 

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?

create table and query tables on hive catalog with s3

### CODE REVIEW

File: docs/spark-connector/spark-catalog-hive.md [hunk 1]
```diff
@@ -70,4 +70,11 @@ Gravitino catalog property names with the prefix `spark.bypass.` are passed to S
...
-:::
\ No newline at end of file
+:::
+
+
+## Storage
+
+### S3
+
+Please refer to [Hive catalog with s3](../hive-catalog-with-s3.md) to set up a hive catalog with s3 storage. To querying the data stored in s3, you need to add s3 secret to the Spark configuration using `spark.sql.catalog.${hive_catalog_name}.fs.s3a.access.key` and `spark.sql.catalog.${iceberg_catalog_name}.s3.fs.s3a.secret.key`. Additionally, download [hadoop aws jar](https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws) [aws java sdk jar](https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-bundle) and place them in the classpath of Spark.
```
Comment (yuqi1129) [inline docs/spark-connector/spark-catalog-hive.md:80]: set up a hive catalog -> set up a Hive catalog
To querying -> to query
please add a ',' before '[aws java sdk jar]'.

Comment (yuqi1129) [reply docs/spark-connector/spark-catalog-hive.md:80]: Can you also add the detail jar name to `hive-catalog-with-s3.md#L211`?

Comment (FANNG1) [reply docs/spark-connector/spark-catalog-hive.md]: done

File: docs/spark-connector/spark-catalog-hive.md [hunk 2]
```diff
@@ -70,4 +70,11 @@ Gravitino catalog property names with the prefix `spark.bypass.` are passed to S
...
-:::
\ No newline at end of file
+:::
+
+
+## Storage
+
+### S3
+
+Please refer to [Hive catalog with s3](../hive-catalog-with-s3.md) to set up a Hive catalog with s3 storage. To query the data stored in s3, you need to add s3 secret to the Spark configuration using `spark.sql.catalog.${hive_catalog_name}.fs.s3a.access.key` and `spark.sql.catalog.${iceberg_catalog_name}.s3.fs.s3a.secret.key`. Additionally, download [hadoop aws jar](https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws) , [aws java sdk jar](https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-bundle) and place them in the classpath of Spark.
```
Comment (yuqi1129) [inline docs/spark-connector/spark-catalog-hive.md:80]: > hadoop-aws) ,

There is a space before ',', please remove it. 

ditto

**Review by yuqi1129 — COMMENTED** [review]
Others LGTM.

File: docs/spark-connector/spark-catalog-hive.md (hunk 2 above)
Comment (FANNG1) [reply docs/spark-connector/spark-catalog-hive.md]: updated

**Review by yuqi1129 — APPROVED** [review]

**Pull request merged**